plugins/nanoleaf/README.md

```diff
@@ -24,5 +27,6 @@ one controller by adding multiple `controller =` lines
 The port to stream to on the controller, range is 1 - 65535.
 
 `<controller IP>-panels = [int]`
-The list of panel IDs to control, each panel is mapped to three DMX512 slots
-(for Red, Green and Blue).
+The list of panel IDs to control, each panel is mapped to three DMX512
+slots (for Red, Green and Blue). 
+Panel IDs are comma separated eg: `321,789,123`
```

Comment:
I've just noticed off the back of #1730 these example values are invalid for the v1 protocol. We'd want say (to show order is irrelevant):
```suggestion
Panel IDs are comma separated eg: `132,178,54`
```